utils.py

```diff
@@ -0,0 +1,63 @@
+from dataclasses import dataclass
+from datetime import datetime
+import os
+
+import numpy as np
+import tskit
+import tszip
+
+def load_tsz_file(date, filename):
+    ts = tszip.decompress(os.path.join("data", filename.format(date)))
+    ts.day0 = datetime.fromisoformat(date)  # Hack: this won't be saved / transferred
+    return ts
+
+def node_spans_parent(ts, min_num_children=None):
```

Comment:
Yeah, sure. I wrote this for my own edification, so it could be placed into tskit later. I'll simplify the code and put the older stuff in the tskit issue you are about to open.